packages/templates/cra-template-web-viewer/template/src/index.tsx

```diff
@@ -29,9 +29,9 @@ if (!process.env.IMJS_AUTH_CLIENT_REDIRECT_URI) {
 }
 
 Auth.initialize({
-  scope: process.env.IMJS_AUTH_CLIENT_SCOPES,
-  clientId: process.env.IMJS_AUTH_CLIENT_CLIENT_ID,
-  redirectUri: process.env.IMJS_AUTH_CLIENT_REDIRECT_URI,
+  scope: process.env.IMJS_AUTH_CLIENT_SCOPES ?? "",
```

Comment:
do you need the fallback to empty strings?

dont the above if statements check if its defined?

Reply:
removed fallback, they are checked earlier